Support Tahoe 2.0 site configuration client (#63)
appsembler/xblock-launchcontainer: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

## [4.0.0] - 2022-09-19
### Added

 - Install `tahoe-sites` and `site-configuration-client` to simulate Tahoe environment.
 - Add support for Tahoe 2.0 site by reading `LAUNCHCONTAINER_WHARF_URL` from Site Config Service `secret`s. Tahoe 1.0 sites will still use `site_values` as usual

> - Breaking change: Drop support for pre-Juniper Tahoe installations. The last version to support Hawthorn is `3.0.0`

Reviewer: ```suggestion
 - Breaking change: Drop support for pre-Juniper Tahoe installations. The last version to support Hawthorn is `3.0.0`
```

> need to complicate the XBlock with caching calls.

Reviewer: ```suggestion
- Do not complicate the XBlock with caching calls.
```

Reviewer: actually I think you can delete this line—it's duplicated below
